app/web/src/newhotness/Explore.vue

```diff
@@ -1105,6 +1118,16 @@ const scrollCurrentTileIntoView = () => {
 const onMapDeselect = () => {
   searchRef.value?.blur();
 };
+
+const storeViewMode = () => {
+  if (!groupRef.value) return;
+
+  if (groupRef.value.isB) {
+    localStorage.setItem(VIEW_MODE_LOCAL_STORAGE_KEY, "grid");
```

Comment:
I would put the `changeSetId` in the key of `setItem`, that way multiple tab usage doesnt create unexpected behavior